bin/install-wp-tests.sh

```diff
@@ -177,8 +178,13 @@ install_db() {
 	if [ $(mysql --user="$DB_USER" --password="$DB_PASS"$EXTRA --execute='show databases;' | grep ^$DB_NAME$) ]
 	then
 		echo "Reinstalling will delete the existing test database ($DB_NAME)"
-		read -p 'Are you sure you want to proceed? [y/N]: ' DELETE_EXISTING_DB
-		recreate_db $DELETE_EXISTING_DB
+		if [ ${OVERRIDE} == "true" ]
+		then
```

Comment:
Das habe ich gemacht, damit ich in meiner lokalen Testumgebung einstellen kann, dass einfach still bei jedem Testdurchlauf die alte Datenbank überschrieben wird. Dann muss ich nicht jedes Mal ein komplett neues System aufsetzen (lassen)